src/test/java/bio/terra/service/profile/ProfileAPIControllerTest.java

```diff
@@ -166,8 +169,7 @@ void testDeleteProfile(
     ResponseEntity<JobModel> entity = apiController.deleteProfile(deleteId, deleteCloudResources);
     // Only check for admin auth if deleteCloudResources is true
     verify(iamService, times(expectedAdminAuthNumberOfInvocations))
-        .verifyAuthorization(
-            eq(user), eq(IamResourceType.DATAREPO), any(), eq(IamAction.CONFIGURE));
+        .verifyAuthorization(eq(user), eq(IamResourceType.DATAREPO), any(), eq(IamAction.DELETE));
```

Comment:
You can eliminate the `any()` here if you save the value you pass to the `when(..getResourceId()).thenReturn(...)` above, e.g.
```
    var applicationId = UUID.randomUUID().toString();
    if (deleteCloudResources) {
      when(applicationConfiguration.getResourceId()).thenReturn(applicationId);
    }
...
    verify(iamService, times(expectedAdminAuthNumberOfInvocations))
        .verifyAuthorization(user, IamResourceType.DATAREPO, applicationId, IamAction.DELETE);
```

Reply:
Good point! Thanks!